packages/app/cypress/e2e/runner/support/spec-loader.ts

```diff
@@ -55,26 +57,13 @@ export function loadSpec (options: LoadSpecOptions): void {
     })
   }, { hasPreferredIde })
 
-  // TODO: investigate why directly visiting the spec will sometimes hang
-  // https://cypress-io.atlassian.net/browse/UNIFY-1154
```

Comment:
were you able to validate that the flake is probably gone after the changes in this file?

Reply:
Yup, haven't seen a failure locally nor after numerous CI builds. This used to fail quite consistently before. There's more info in the PR description, but #20558 corrected the issue that was previously causing the hang.